### PR TITLE
Fixed eslint `ENOENT` (`Error NO ENTry`) error 

### PR DESCRIPTION
**Pull request by joelerego**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
     "parser": "@typescript-eslint/parser",
-    "extends": [
-        "eslint:recommended",
-        "plugin:@typescript-eslint/recommended",
-        "react-app",
-        "prettier"
-    ],
-    "plugins": ["@typescript-eslint", "prettier"],
+    "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+    "plugins": ["@typescript-eslint", "react", "react-hooks"],
```
Comment (itssamuelrowe) [inline .eslintrc.json:5]: Is it advisable to include `react` and `react-hooks` here?

**Review by itssamuelrowe — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (joelerego) [reply .eslintrc.json:5]: Yes, without them you can't use ignore rules such as exhaustive dependency lists in `useEffect` hooks.

Comment (itssamuelrowe) [reply .eslintrc.json:5]: Since the configuration is in the root, will including `react` have an effect on packages that don't use React?

**Review by itssamuelrowe — APPROVED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (itssamuelrowe) [reply .eslintrc.json:5]: Alright.

**Pull request merged**